apis/apps/v1alpha1/servicedescriptor_types.go

```diff
@@ -68,6 +68,11 @@ type ServiceDescriptorSpec struct {
 	//
 	// +optional
 	Auth *ConnectionCredentialAuth `json:"auth,omitempty"`
+
+	// Specifies the pod FQDNs of the external service.
+	//
+	// +optional
+	PodFQDNs *CredentialVar `json:"podFQDNs,omitempty"`
```

Comment:
This field is unnecessary. Users can configure any string in the endpoint, including pod FQDN type addresses.

Reply:
@weicao 
There is a need to include references to PodFQDNs in serviceRef, for example:
1. When I use scripts or fluxcd to batch deploy clusters instead of manually deploying, it is a bit troublesome to build FQDN through expression
2. When I scale the referenced cluster, It can dynamically generate FQDN based on the current replicas, thereby reducing the risk of manual changes
3. 
In summary, it is recommended that the community add references to FNQDs in serviceRef

Reply:
If we don't define it, we can't explicitly reference the FQDN of service-ref in vars.